### PR TITLE
Add image of expiration notification

### PR DESCRIPTION
**Pull request by johnludwigm**
Still needs image

### CODE REVIEW

**Comment by PRMerger20** [comment]
@johnludwigm : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@johnludwigm : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/paginated-reports/paginated-reports-faq.md [hunk 1]
```diff
@@ -174,7 +174,9 @@ Here's something to keep in mind when working with DateTime fields in paginated
 
 - Currently there are some globalization limitations related to DateTime parameters. All DateTime parameters in the Power BI service are fetched in US format (MM/DD/YYYY) regardless of how you design the DataTime in Power BI Report Builder.
 
-When viewing paginated reports in the Power BI service:
+When viewing paginated reports in the Power BI service, sessions may time out, presenting the user with the notification below:
```
Comment (maggiesMSFT) [inline powerbi-docs/paginated-reports/paginated-reports-faq.md:177]:
```suggestion
When viewing paginated reports in the Power BI service, sessions may time out, presenting the user with the following notification:
```

**Comment by PRMerger13** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**